en/contributing/test.md

```diff
@@ -10,10 +10,10 @@ This topic shows how to run the existing tests.
 
 ## Running Unit Tests
 
-To run all unit tests:
+To run all unit tests: 
 
 ```
-make run_unit_tests
+make run_unit_tests 
```

Comment:
@julianoes This runs only core unit tests. I tried `make run_unit_tests EXTERNAL_DIR=../DroneCore_Extensions` but no luck.

How do we run unit tests for external libraries?

Reply:
Thanks, that's correct, that's currently broken.

Reply:
Is there an issue I can track this with?

Reply:
Or more specifically, what is the syntax, and I can update the docs even if it doesn't work yet.

Reply:
No, it's just not working right now and there is a lot of churn around this given the gRPC changes so not worth it to fix right now.